CHANGELOG.md

```diff
@@ -4,17 +4,29 @@
 
 * ...
 
+
+## 4.2.1     2023-05-17
+
+* Parse: Fix `ALTER INDEX my_index_name` to return non-nil `tables` ([#285](https://github.com/pganalyze/pg_query/pull/285))
+* Parse: Detect tables in a SELECT INTO clause as DDL tables ([#281](https://github.com/pganalyze/pg_query/pull/281))
+* Add support for Ruby 3.2 ([#283](https://github.com/pganalyze/pg_query/pull/283))
```

Comment:
Looks like we removed Ruby 2.5 from CI with this PR, but we still say that it's supported version. Should we make it clear that we dropped the support of 2.5, also update README?

https://github.com/pganalyze/pg_query#supported-ruby-versions